hogben/simulate.py

```diff
@@ -8,9 +10,32 @@
 import numpy as np
 
 
-def simulate_magnetic(sample, angle_times, scale=1, bkg=5e-7, dq=2,
-                      mm=True, mp=True, pm=True, pp=True, angle_scale=0.7,
-                      directbeam_path=None):
+def DB_path(instrument: str, polarised: bool = False) -> str:
+    """Returns the filepath of the correct direct beam file for the instrument being used
+
+    Args:
+        instrument: The name of the instrument the experiement will be performed on
```

Comment:
Describing the valid options in the docstring could be useful. e.g. "valid non-polarised instruments are ['OFFSPEC', 'SURF', 'POLREF'], valid polarised instruments are ['OFFSPEC']" or something similar